pkg/apis/batch/validation/validation.go

```diff
@@ -546,8 +546,8 @@ func ValidateJobTemplateSpec(spec *batch.JobTemplateSpec, fldPath *field.Path, o
 
 type JobValidationOptions struct {
 	apivalidation.PodValidationOptions
-	// Allow Job to have the annotation batch.kubernetes.io/job-tracking
-	AllowTrackingAnnotation bool
+	// Allow Job to not have the annotation batch.kubernetes.io/job-tracking
```

Comment:
```suggestion
	// Allow Job to not to have the annotation batch.kubernetes.io/job-tracking
```

Reply:
ok, I didn't like either option, so I changed to logic to be `RequireTrackingAnnotation=true` by default, and false if the old job didn't have it.